java/spring-webflux/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.5</version>
+		<relativePath/>
+	</parent>
+	<groupId>framework</groupId>
+	<artifactId>benchmark-webflux</artifactId>
+	<version>1.0.0</version>
+	<name>benchmark-webflux</name>
+	<description>Benchmark for Spring WebFlux</description>
+	<properties>
+		<java.version>17</java.version>
```

Comment:
spring is not yet compatible with 21 ?

Reply:
Spring Boot version 3.2, scheduled for release on November 23, will be fully compatible with Java 21. This update includes support for Virtual Threads, an exciting feature that enhances performance and efficiency, particularly in benchmark scenarios.  I think it is better to change it to Java 21 after November 23 and we also enable Virtual Threads feature.

Reply:
could you create an issue for that ? or wait until the bot make an update PR ?